### DISCUSSION TITLE
Get the message of useLazyFetch error

### DISCUSSION THREAD

**Discussion by ahoiroman** [nuxt/nuxt#17501] [Questions]
Hello everybody,

I am using useLazyFetch to fetch data from an api endpoint like this:

```
    const {data, error} = useLazyFetch(`/backend/test`, {server: false,});

    console.log(error.value.data) //returns 
```

This is working out and returns something like:

```
{
  "message": "...",
  "errors": {
    employment.verification: [
      //...
    ]
  }
}
```

But my IDE reports an typescript error:

> TS2339: Property 'data' does not exist on type 'true | Error'.   Property 'data' does not exist on type 'true'.

<img width="724" alt="Bildschirmfoto 2022-06-01 um 11 36 44@2x" src="https://user-images.githubusercontent.com/23051150/171375244-d961ac7f-0e59-46f9-9c35-67c06798280f.png">

What am I doing wrong?

**Comment by wobsoriano** [comment]
the error only provides a Boolean on the client-side, so if you need the server error on the client-side, you would save it to a `useState`. Here's my implementation:

https://github.com/nuxt/framework/issues/2122#issuecomment-1131170846
